pkgs/tools/misc/pridecat/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, stdenv, fetchgit }:
+
+let commit = "92396b11459e7a4b5e8ff511e99d18d7a1589c96";
```

Comment:
Why are you using a let-binding for a literal that you only use once?

Reply:
this was adapted from a local package that used commit as the version